src/components/Container.jsx

```diff
@@ -15,25 +15,25 @@ const PersonData = {
 };
 
 function Container() {
-  const [team, setTeam] = useState(data);
+  const [team, setTeam] = useState('');
   const [active, setActive] = useState(PersonData.All);
   const [searchTerm, setSearchTerm] = useState('');
+  const [minValue, setMinValue] = useState('');
+  const [maxValue, setMaxValue] = useState('');
+
+  const handleMinValue = (event) => {
+    if (event.target.value === '') setMinValue('');
+    else setMinValue(event.target.value);
+  };
+
+  const handleMaxValue = (event) => {
+    if (event.target.value === '') setMaxValue('');
+    else setMaxValue(event.target.value);
```

Comment:
If value equals to empty string then setMaxValue(null) else setMaxValue(Number(event.target.value))